cloudify_agent/resources/script/linux.sh.template

```diff
@@ -26,8 +26,13 @@ configure_agent_installation()
 
    # Prepare and create agent directory and provide write permission to users belongs to cfyagent group
    sudo mkdir -p {{ conf.agent_dir }}
-   sudo chgrp -R cfyagent $(dirname {{ conf.agent_dir }})
-   sudo chmod -R g+w $(dirname {{ conf.agent_dir }})
+   # Currently agent data is expected to live in ~user/<agent name>, so we need to allow cfyagent group access to there,
+   # but without breaking everything else in the homedir (e.g. we shouldn't mess with permissions in .ssh)
+   sudo chgrp cfyagent $(dirname {{ agent_dir }})
+   sudo chmod g+x $(dirname {{ agent_dir }})
+   # Ensure that the cfyagent user can read and write and files and directories, and access any dirs
+   sudo chgrp -R cfyagent {{ agent_dir }}
+   sudo chmod -R g+rwX {{ agent_dir }}
```

Comment:
@geokala you missed the `conf.` this is different from the other place for linux-download.sh